src/main/java/battin/preston/RecordNotPresent.java

```diff
@@ -0,0 +1,13 @@
+package battin.preston;
+
+/**
+ * Created by prestonbattin on 2/7/17.
+ */
+public class RecordNotPresent extends Exception {
```

Comment:
usually Exceptions are defined with a name that ends in "Exception"  -- makes it easier to spot them when looking through your codebase.